apps/cookbook/src/app/page/side-nav/side-nav.component.scss

```
@@ -92,10 +92,8 @@ hr {
   margin: utils.size('xxxs'), 0;
   border: solid 1px utils.get-color('light');
   border-bottom: none;
-  margin-block-start: $divider-spacing;
-  margin-block-end: $divider-spacing;
-  margin-inline-start: $divider-inset;
-  margin-inline-end: $divider-inset;
+  margin-block: $divider-spacing $divider-spacing;
+  margin-inline: $divider-inset $divider-inset;
```

Consider simplifying further (same values):
```suggestion
  margin-block: $divider-spacing;
  margin-inline: $divider-inset;
```